nemo/collections/vlm/neva/data/config.py

```diff
@@ -31,15 +31,15 @@ class DataConfig:
 @dataclass
 class ImageDataConfig(DataConfig):
     media_type: str = "image"
-    media_token: MultiModalToken = ImageToken
+    media_token: MultiModalToken = ImageToken()
```

Comment:
can you double check if dataclass can initialize a class instance as default value? I thought you have to use default factory